module-apps/application-meditation/ApplicationMeditation.cpp

```diff
@@ -46,10 +69,29 @@ namespace app
 
     void ApplicationMeditation::createUserInterface()
     {
-        windowsFactory.attach(app::window::name::meditation_main_window,
-                              [](ApplicationCommon *app, const std::string &name) {
-                                  return std::make_unique<gui::MeditationWindow>(app);
-                              });
+        windowsFactory.attach(
+            app::window::name::meditation_main_window, [&](ApplicationCommon *app, const std::string &name) {
+                const auto durationInitValue = utils::getNumericValue<std::int32_t>(
```

Comment:
By the design `utils::getNumericValue` returns 0 in case of error. At the same time, 0 can be a valid value too. Do you think it can be a possible issue here?

Reply:
Such an error should not happen, but if it does, there are also validations of the set parameters. In the extreme case, the user will have the "chime interval" set to 0.